content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -65,6 +65,8 @@ order of seconds.
 
 ## 503 errors after setting destination rule
 
+You should only see this error when you disable [automatic mutual](docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) TLS during install.
```

Comment:
```suggestion
{{< tip >}}
You should only see this error if you disabled [automatic mutual TLS](docs/tasks/security/authentication/authn-policy/#auto-mutual-tls) during install.
{{< /tip >}}
```

Reply:
thanks! i was thinking there indeed some cutomized tag can be used for highlight, but can't rembmer the name!